features/step_definitions/scc.rb

```diff
@@ -105,7 +105,7 @@
     @result = _admin.cli_exec(_command, **_opts)
     @result[:success]
   }
-  if @result[:success]
+  if @result[:success] && !no_teardown
```

Comment:
@liangxia , could you make sure scenario is tagged with `@ugrade_prepare` in case no teardown is selected?

Reply:
@akostadinov  https://github.com/openshift/verification-tests/pull/1444